README.md

```diff
@@ -158,6 +158,7 @@ These are the keys you can specify:
   --> `all-contributors-cli`. Mandatory.
 * `repoType`: Type of repository. Must be either `github` or `gitlab`. Default: `github`.
 * `repoHost`: Points to the repository hostname. Change it if you use a self hosted repository. Default: `https://github.com` if `repoType` is `github`, and `https://gitlab.com` if `repoType` is `gitlab`.
+* `private_token`: The personal access token to to authenticate with the GitLab API. Offer it if you use a self hosted repository. Default: `''`.
```

Comment:
Could this follow the existing camel casing convention? It should be `privateToken`.